core/model/src/main/java/org/eclipse/sw360/antenna/model/artifact/facts/java/MavenCoordinates.java

```diff
@@ -61,6 +61,17 @@ public String getVersion() {
         return version;
     }
 
+    @Override
+    public String getType() {
+        return "maven";
+    }
+
+    @Override
+    public String getPurl() {
+        return String.format("pkg:%s/%s/%s%s", percentEncoding(getType()), percentEncoding(getGroupId()),
```

Comment:
The `type` must not be percent-encoded, see https://github.com/package-url/purl-spec#rules-for-each-purl-component.